pkg/collector/corechecks/net/network/network.go

```diff
@@ -200,6 +217,14 @@ func (c *NetworkCheck) isDeviceExcluded(deviceName string) bool {
 
 func submitInterfaceMetrics(sender sender.Sender, interfaceIO net.IOCountersStat) {
 	tags := []string{fmt.Sprintf("device:%s", interfaceIO.Name), fmt.Sprintf("device_name:%s", interfaceIO.Name)}
+	speedVal, err := readIntFile(fmt.Sprintf("/sys/class/net/%s/speed", interfaceIO.Name), filesystem)
```

Comment:
I noticed we might be missing this proc_dev_path read the python check does: https://github.com/DataDog/integrations-core/blob/3fd29a6cf3ea69b09b8deec35bbce0e25b9578d3/network/datadog_checks/network/check_linux.py#L116-L131